### PR TITLE
Adding Test Cases

### PR DESCRIPTION
**Pull request by devendragovil**
Adding test cases for the neighborhood module.

### CODE REVIEW

**Comment by devendragovil** [comment]
@ninamiolane Requesting your review. Docs deployment failed, but I think it has nothing to do with the code changes.

File: test/test_neighborhood.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        """Testing if right assertion is raised for incorrect type."""
+        with pytest.raises(TypeError) as e:
+            tex.neighborhood.neighborhood_from_complex(1)
+
+        assert (
+            str(e.value)
+            == """Input Complex can only be a Simplicial, Cell or Combinatorial Complex."""
+        )
+
+    def test_matrix_dimensions_cellcomplex(self):
```
Comment (ninamiolane) [inline test/test_neighborhood.py:22]: According to the unit-test framework, the name of the test function has to be test_[name of the function being tested].

here: test_neighborhood_from_complex.

**Review by ninamiolane — CHANGES_REQUESTED** [review]
Nice, thanks! Very important to test this module indeed. I have made one minor, yet very important, comment on the naming of test functions.

Why is the docs deployment failing?

**Comment by devendragovil** [comment]
@ninamiolane  Have made the requested changes.

**Comment by devendragovil** [comment]
@ninamiolane 
I am trying to resolve the doc deployment problem. One similar issue seems to relate to insufficient permissions with access tokens: https://github.com/JamesIves/github-pages-deploy-action/issues/922

**Comment by ninamiolane** [comment]
I see for the docs issue! Hm, let's just try to merge the PR then 🤞

**Pull request merged**